packages/overlay/src/integrations/sentry/components/traces/TraceListFilter.tsx

```diff
@@ -0,0 +1,117 @@
+import { ReactComponent as X } from '~/assets/cross.svg';
+import { ReactComponent as Search } from '~/assets/search.svg';
+
+import { useCallback, useMemo } from 'react';
+import { Badge } from '~/ui/badge';
+import { Button } from '~/ui/button';
+import { Input } from '~/ui/input';
+import { getSpotlightContainer } from '~/utils/dom';
+import { type FilterConfigs } from '../../hooks/useTraceFiltering';
+import { FilterDropdown } from '../shared/FilterDropdown';
+
+interface TraceListFilterProps {
+  searchQuery: string;
+  setSearchQuery: (query: string) => void;
+  activeFilters: string[];
+  setActiveFilters: React.Dispatch<React.SetStateAction<string[]>>;
+  filterConfigs: FilterConfigs;
+}
+
+export default function TraceListFilter({
+  searchQuery,
+  setSearchQuery,
+  activeFilters,
+  setActiveFilters,
+  filterConfigs,
+}: TraceListFilterProps) {
+  const spotlightContainer = getSpotlightContainer();
+
+  const handleFilterChange = useCallback(
+    (value: string, checked: boolean) => {
+      if (checked) {
+        setActiveFilters(prev => [...prev, value]);
```

Comment:
being able to select more than 1 time filter could be confusing. If you selected "last minute" and then "last 5 minutes". not sure if it will be as simple as doing something like:
```typescript
...
const isTimeFilter = Object.values(TIME_FILTER_VALUES).includes(value); 
...
if (checked){
  if (isTimeFilter){
  setActiveFilters(...) // filter to remove any filter included in TIME_FILTER_VALUES and include the new value
  }
}
``` 

If that works, maybe it's a good idea to do it, if it requires more work maybe we can leave it as it is